website/docs/tutorials/embed_linear_genome_view/09_conclusion.md

```diff
@@ -0,0 +1,264 @@
+        },
+        location: '1:100,987,269..100,987,368',
+        defaultSession: {
+          name: 'my session',
+          view: {
+            id: 'linearGenomeView',
+            type: 'LinearGenomeView',
+            tracks: [
+              {
+                id: 'IpTYJKmsp',
+                type: 'ReferenceSequenceTrack',
+                configuration: 'GRCh38-ReferenceSequenceTrack',
+                displays: [
+                  {
+                    id: 's877wHWtzD',
+                    type: 'LinearReferenceSequenceDisplay',
+                    configuration:
+                      'GRCh38-ReferenceSequenceTrack-LinearReferenceSequenceDisplay',
+                  },
+                ],
+              },
+            ],
+          },
+        },
+      })
+      function navTo(event) {
+        state.session.view.navToLocString(event.target.dataset.location)
+      }
+      const buttons = document.getElementsByTagName('button')
+      for (const button of buttons) {
+        if (button.dataset.type === 'gene_button') {
+          button.addEventListener('click', navTo)
+        }
+      }
+      const textArea = document.getElementById('session')
+      document.getElementById('showsession').addEventListener('click', () => {
+        textArea.innerHTML = JSON.stringify(state.session, undefined, 2)
+      })
+      render(
+        createElement(JBrowseLinearGenomeView, { viewState: state }),
+        document.getElementById('jbrowse_linear_genome_view'),
+      )
+    </script>
+    <button id="showsession">Show current session</button>
+    <textarea id="session" name="session" rows="20" cols="80"></textarea>
+    <p>updates:</p>
+    <textarea
+      id="update"
+      name="update"
+      rows="5"
+      cols="80"
+      wrap="off"
+    ></textarea>
+  </body>
+</html>
+```
+
+  </p>
+</details>
+
+<details>
+  <summary>assembly.js</summary>
+  <p>
+
+```javascript title="assembly.js"
+export default {
+  name: 'GRCh38',
+  sequence: {
+    type: 'ReferenceSequenceTrack',
+    trackId: 'GRCh38-ReferenceSequenceTrack',
+    adapter: {
+      type: 'BgzipFastaAdapter',
+      fastaLocation: {
+        uri:
+          'http://ftp.ensembl.org/pub/release-100/fasta/homo_sapiens/dna_index/Homo_sapiens.GRCh38.dna.toplevel.fa.gz',
+      },
+      faiLocation: {
+        uri:
+          'http://ftp.ensembl.org/pub/release-100/fasta/homo_sapiens/dna_index/Homo_sapiens.GRCh38.dna.toplevel.fa.gz.fai',
+      },
+      gziLocation: {
+        uri:
+          'http://ftp.ensembl.org/pub/release-100/fasta/homo_sapiens/dna_index/Homo_sapiens.GRCh38.dna.toplevel.fa.gz.gzi',
+      },
+    },
+  },
+  aliases: ['hg38'],
+  refNameAliases: {
+    adapter: {
+      type: 'RefNameAliasAdapter',
+      location: {
+        uri:
+          'http://s3.amazonaws.com/jbrowse.org/genomes/GRCh38/GRCh38.aliases.txt',
+      },
+    },
+  },
+}
+```
+
+  </p>
+</details>
+
+<details>
+  <summary>assembly.js</summary>
```

Comment:
this should be `<summary>tracks.js</summary>`

Reply:
fixed!